addons/survey/wizard/survey_invite.py

```diff
@@ -66,6 +65,16 @@ def _get_default_author(self):
     survey_users_login_required = fields.Boolean(related="survey_id.users_login_required", readonly=True)
     survey_users_can_signup = fields.Boolean(related='survey_id.users_can_signup')
     deadline = fields.Datetime(string="Answer deadline")
+    send_email = fields.Boolean(compute="_compute_send_email",
```

Comment:
Could you add a specific label?

I don't like the "automatic" mechanism, as it prevents from freely renaming the field.